doc/news/changes/minor/20190807ImbertGerard.txt

```diff
@@ -0,0 +1,3 @@
+changed the comment to the hyper_L function
+ 
+(Lise-Marie Imbert-Gerard, August 7, 2019)
```

Comment:
Please use the same format as the other files in the same folder (date format and use ``<br>``, ...)